server/public/model/utils.go

```diff
@@ -531,6 +531,16 @@ func ToJSON(v any) []byte {
 	return b
 }
 
+// FromJSON decodes an arbitrary object
+func ObjectFromJSON[T any](data io.Reader, obj T, maxBytes int64) error {
+	lr := io.LimitedReader{N: maxBytes, R: data}
+	err := json.NewDecoder(&lr).Decode(&obj)
```

Comment:
We are already passing the pointer from the call site. Do we need to pass the pointer to that again here?